pkg/scheduler/cache/event_handlers.go

```diff
@@ -286,11 +284,7 @@ func (sc *SchedulerCache) AddNode(obj interface{}) {
 	sc.Mutex.Lock()
 	defer sc.Mutex.Unlock()
 
-	err := sc.addNode(node)
-	if err != nil {
-		klog.Errorf("Failed to add node %s into cache: %v", node.Name, err)
-		return
-	}
+	sc.addNode(node)
```

Comment:
let's keep oldNode, if not used, try `_`.

Reply:
recoveried